src/discrete-layer/components/map-container/layers-footprints.tsx

```diff
@@ -0,0 +1,112 @@
+import React, { useEffect, useState } from 'react';
+import { observer } from 'mobx-react-lite';
+import { CesiumGeojsonLayer, CesiumColor } from '@map-colonies/react-components';
+import { Feature, FeatureCollection, Geometry, Polygon } from 'geojson';
+import polygonToLine from '@turf/polygon-to-line';
+import bbox from '@turf/bbox';
+import bboxPolygon from '@turf/bbox-polygon';
+import { useStore } from '../../models/rootStore';
+
+export const LayersFootprints: React.FC = observer(() => {
+  const { discreteLayersStore } = useStore();
+  const [layersFootprints, setlayersFootprints] = useState<FeatureCollection>();
+
+  // REMARK: Layers footprint boundingboxes  
+  useEffect(() => {
+    if (discreteLayersStore.layersImages) {
+      const footprintsCollection: FeatureCollection = {
+        type: 'FeatureCollection',
+        features: []
+      }
+      const footPrintsFeaturesArray = discreteLayersStore.layersImages.map((layer) => {
+        let geometry: Geometry = layer.geojson as Geometry;
+        switch(layer.geojson?.type){
+          case 'Polygon':
+            geometry = (polygonToLine(geometry as Polygon) as Feature).geometry;
+            break;
+          case 'MultiPolygon':
+            //get bbox of feture, then convert it to polygon and finally get linestring of polygon
+            geometry = (polygonToLine(bboxPolygon(bbox(geometry))) as Feature).geometry;
```

Comment:
![image](https://user-images.githubusercontent.com/18442671/103364507-b2486100-4ac6-11eb-823b-1bcf7407c810.png)
why the conversion to BBOX when Turf can handle it?

Reply:
For this case I need only outer Feature which is presented as single LineString geojson Feature.

In this file present two kind of visualizations: **outer BBOX** (Feature<LineString>) and **geoJSON feature as is** only one of them should be chosen @shaziri want a geoJSON as is with different fill colors.

Think at this time no action item, code remained unchanged.